modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,115 @@
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Manual`.
+
+. To generate the manifests, run the following command from the directory that contains the installation program:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
+----
+
+. From the directory that contains the installation program, obtain details of the {product-title} release image that your `openshift-install` binary is built to use:
++
+[source,terminal]
+----
+$ openshift-install version
+----
++
+.Example output
+[source,terminal]
+----
+release image quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64
+----
+
+. Extract the `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal]
+----
+$ oc adm release extract --cloud=ibmcloud --credentials-requests $RELEASE_IMAGE \
+    --to=<path_to_directory_with_list_of_credentials_requests> <1>
+----
+<1> The directory to extract the credential requests to.
++
+This command creates a YAML file for each `CredentialsRequest` object.
++
+.Sample `CredentialsRequest` object
++
+[source,yaml]
+----
+  apiVersion: cloudcredential.openshift.io/v1
+  kind: CredentialsRequest
+  metadata:
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: openshift-image-registry-ibmcos
+    namespace: openshift-cloud-credential-operator
+  spec:
+    secretRef:
+      name: installer-cloud-credentials
+      namespace: openshift-image-registry
+    providerSpec:
+      apiVersion: cloudcredential.openshift.io/v1
+      kind: IBMCloudProviderSpec
+      policies:
+      - attributes:
+        - name: serviceName
+          value: cloud-object-storage
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+        - crn:v1:bluemix:public:iam::::role:Operator
+        - crn:v1:bluemix:public:iam::::role:Editor
+        - crn:v1:bluemix:public:iam::::serviceRole:Reader
+        - crn:v1:bluemix:public:iam::::serviceRole:Writer
+      - attributes:
+        - name: resourceType
+          value: resource-group
+        roles:
+        - crn:v1:bluemix:public:iam::::role:Viewer
+----
+
+. Create the service ID for each credential request, assign the policies defined, create an API key in IBM Cloud, and generate the secret:
++
+[source,terminal]
+----
+$ ccoctl ibmcloud create-service-id \
+    --credentials-requests-dir <path_to_directory_with_list_of_credentials_requests> \ <1>
+    --name <name> \ <2>
+    --resource-group-name <resource_group_name> <3>
+----
+<1> The directory with the generated list of credential requests.
+<2> The name of the {product-title} cluster.
+<3> Optional: The resource group name of the cluster.
+
+.Verification
+
+* Ensure that the appropriate secrets were generated in your cluster's `manifests` directory.
```

Comment:
The path `path_to_directory_with_list_of_credentials_requests` to me, is ambiguous.  It appears that `oc` generates this path and `ccoutl` acts upon the output.  It is not clear to me if this path is `<install-dir>/manifests` or another path.  It appears it is another path, the verification step implies otherwise.  It might be helpful to clarify how the generated manifests land in the cluster's `manifests` directory.

Reply:
Hi @rvanderp3, I agree, the name could be confusing, that path can be an existing one or not, the `ccoctl` binary will create it based on the name you used there, that path can be also external to the installation folder, the critical step is the `ccoctl ibmcloud create-service-id` that should point to this generated path and also to the installer folder with `--output-dir <installation_directory>` in order to inject there the corresponding manifest files.
NOTE: we are missing the `--output-dir` parameter there, I've already reported that to @codyhoag